tests/sentry/api/test_event_search.py

```diff
@@ -653,6 +655,107 @@ def test_invalid_aggregate_column_with_duration_filter(self):
         with self.assertRaises(InvalidSearchQuery, regex="not a duration column"):
             parse_search_query("avg(stack.colno):>500s")
 
+    def test_numeric_measurements_filter(self):
+        # NOTE: can only filter on integers right now
+        assert parse_search_query("measurements.size:10") == [
+            SearchFilter(
+                key=SearchKey(name="measurements.size"),
+                operator="=",
+                value=SearchValue(raw_value=10),
+            )
+        ]
+
+        assert parse_search_query("measurements.size:>10") == [
+            SearchFilter(
+                key=SearchKey(name="measurements.size"),
+                operator=">",
+                value=SearchValue(raw_value=10),
+            )
+        ]
+
+        assert parse_search_query("measurements.size:<10") == [
```

Comment:
Would these conditions also accept float values?

Reply:
Ah yes, I'll update these tests to also reflect the ability to filter on float values.

Reply:
Addressed in 531ffba